analyzer/CommandLine.Analyzer.Test/CommandLine.Analyzer.Test.csproj

```diff
@@ -1,14 +1,16 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFrameworks>netcoreapp2.0;</TargetFrameworks>
```

Comment:
Any particular reason to use the 'multi-target friendly' property here?

Reply:
Appveyor allows running tests against different target frameworks for free

Reply:
True, but I think VS does better with the single `TargetFramework` property. And this is not actually used, right? Since the analyzer tests are running on netcoreapp2.0 only.